app/views/partials/multi_factor_authentication/_mfa_selection.html.erb

```diff
@@ -2,19 +2,15 @@
   <%= check_box_tag(
         'two_factor_options_form[selection][]',
         option.type,
-        option.type == 'phone' && flash[:phone_error].present? && IdentityConfig.store.kantara_2fa_phone_restricted,
+        false,
         disabled: option.disabled?,
         checked: option.disabled?,
         class: 'usa-checkbox__input usa-checkbox__input--tile',
         id: "two_factor_options_form_selection_#{option.type}",
       ) %>
   <%= label_tag(
         "two_factor_options_form_selection_#{option.type}",
-        class: [
-          option.type == 'phone' && flash[:phone_error] && 'checkbox__invalid' && IdentityConfig.store.kantara_2fa_phone_restricted,
```

Comment:
There's a lingering reference to this `checkbox__invalid` class in the spec that we should probably remove as well:

https://github.com/18F/identity-idp/blob/c545e834361d5bc9bf8f524264ffd2745590292c/spec/views/partials/multi_factor_authentication/_mfa_selection.html.erb_spec.rb#L23-L25